src/univariate/continuous/arcsine.jl

```diff
@@ -67,8 +67,8 @@ mode(d::Arcsine) = d.a
 modes(d::Arcsine) = [d.a, d.b]
 
 var(d::Arcsine) = abs2(d.b - d.a) / 8
-skewness(d::Arcsine{T}) where {T<:Real} = zero(T)
-kurtosis(d::Arcsine{T}) where {T<:Real} = -T(3/2)
```

Comment:
Yeah, this shouldn't be constrained here at all. If you can make an `::Arcsine{T}`, `kurtosis` has no business checking that `T` is "valid".

Reply:
yup this was a common repeated mistake in the code base